pyfr/solvers/baseadvec/elements.py

```diff
@@ -99,3 +99,43 @@ def filter_soln(uin):
                 return self._be.ordered_meta_kernel([mul, copy])
 
             kernels['filter_soln'] = filter_soln
+
+        
+        shock_capturing = self.cfg.get('solver', 'shock-capturing', 'none')
+        if shock_capturing == 'entropy-filter':
+            tags = {'align'}
+
+            self.entmin = self._be.matrix((1, self.neles), 
+                                           tags=tags, extent='entmin')
```

Comment:
Check if these extents are really needed.  The rule is that an extent is needed for any matrix you want to make part of a view.